importer.js

```diff
@@ -78,7 +78,19 @@ class Importer {
       stripUnknown: true,
     });
 
-    this.config.csv.columns = true;
+    this.config.csv.columns = (cols) => { // callback for checking columns names in csv
+      Object.keys(this.config.mapping.fieldSchema).forEach((key) => {
+        if (cols.indexOf(key) < 0) // if key doesn't exist in cols array
+        {
+          console.error('Error: there is no column named ' + key + ' in ' + inputFile);
+          console.log('column names: ' + cols)
```

Comment:
It's strange that you write something to `console.error` and then to `console.log` and then finish the process. 
I believe that you wanted to write to `console.error` again